From: Carsten Schoenert <c.schoenert@t-online.de>
Date: Sun, 29 Jun 2014 13:46:17 +0200
Subject: adjusting bin/server script to debian used folders

There are some hardcoded calls and variables inside the script that
doesn't suite with the debian style. Currently just a hack to make the
stuf work, needs more improvements!
---
 bin/server | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/bin/server b/bin/server
index f9cd75e..5afd16b 100755
--- a/bin/server
+++ b/bin/server
@@ -4,22 +4,26 @@
 # resolve symlinks
 #
 
+#set -x
+
 PRG=$0
 
 while [ -h "$PRG" ]; do
     ls=`ls -ld "$PRG"`
     link=`expr "$ls" : '^.*-> \(.*\)$' 2>/dev/null`
     if expr "$link" : '^/' 2> /dev/null >/dev/null; then
-	PRG="$link"
+        PRG="$link"
     else
-	PRG="`dirname "$PRG"`/$link"
+        PRG="`dirname "$PRG"`/$link"
     fi
 done
 
 progdir=`dirname "$PRG"`
 APPNAME=`basename "$PRG"`
 
-if [ -f "$progdir/../etc/$APPNAME".conf ] ; then
+if [ -f /etc/gridvis-server/server.conf ]; then
+    . /etc/gridvis-server/server.conf
+elif [ -f "$progdir/../etc/$APPNAME".conf ]; then
     . "$progdir/../etc/$APPNAME".conf
 fi
 
@@ -76,7 +80,8 @@ absolutize_paths() {
     done
 }
 
-clusters=`(cat "$progdir/../etc/$APPNAME".clusters; echo) | readClusters | absolutize_paths | tr '\012' ':'`
+#clusters=`(cat "$progdir/../etc/$APPNAME".clusters; echo) | readClusters | absolutize_paths | tr '\012' ':'`
+clusters=`(cat "/etc/gridvis-server/$APPNAME".clusters; echo) | readClusters | absolutize_paths | tr '\012' ':'`
 
 if [ ! -z "$extra_clusters" ] ; then
     clusters="$clusters:$extra_clusters"
